install/operator.yaml

```diff
@@ -141,6 +141,8 @@ rules:
   - configmaps
   verbs:
   - get
+  - list
```

Comment:
Do we actually need these additional permissions? I don't see them used anywhere.

Reply:
This is the error thrown by controller when the permission is not provided:
```
{"level":"info","ts":1649794189.9426928,"logger":"webhooks","msg":"default","name":"cockroachdb"}
{"level":"info","ts":1649794189.98251,"logger":"controller.CrdbCluster","msg":"Running action with name: VersionCheckerAction","CrdbCluster":"default/cockroachdb","ReconcileId":"xKiq62WjKvTaPT2AcJnDFf"}
{"level":"warn","ts":1649794189.9826143,"logger":"controller.CrdbCluster","msg":"starting to check the logging config provided","CrdbCluster":"default/cockroachdb","ReconcileId":"xKiq62WjKvTaPT2AcJnDFf"}
E0412 20:09:49.990532       1 reflector.go:138] external/io_k8s_client_go/tools/cache/reflector.go:167: Failed to watch *v1.ConfigMap: failed to list *v1.ConfigMap: configmaps is forbidden: User "system:serviceaccount:cockroach-operator-system:cockroach-operator-sa" cannot list resource "configmaps" in API group "" at the cluster scope
E0412 20:09:50.984180       1 reflector.go:138] external/io_k8s_client_go/tools/cache/reflector.go:167: Failed to watch *v1.ConfigMap: failed to list *v1.ConfigMap: configmaps is forbidden: User "system:serviceaccount:cockroach-operator-system:cockroach-operator-sa" cannot list resource "configmaps" in API group "" at the cluster scope
E0412 20:09:52.917644       1 reflector.go:138] external/io_k8s_client_go/tools/cache/reflector.go:167: Failed to watch *v1.ConfigMap: failed to list *v1.ConfigMap: configmaps is forbidden: User "system:serviceaccount:cockroach-operator-system:cockroach-operator-sa" cannot list resource "configmaps" in API group "" at the cluster scope
```